### PR TITLE
Update upgrade-postgres.rst

### LINKED ISSUE #5545: "Upgrading Postgres for the CKAN 2.6 release" creates non-SQL ".sql" files
**CKAN version**
2.9

**Describe the bug**
[The Upgrading section of the Postgres for the CKAN 2.6 release page of the Maintainer's guide](https://docs.ckan.org/en/2.9/maintaining/upgrading/upgrade-postgres.html#upgrading) is misleading. The first variant of step 12 instructs to create dump files named _backup_ckan.sql_ and _backup_datastore.sql_. Yet these are not SQL files, due to usage of format "c" instructed by `-Fc`.

**Steps to reproduce**
This is not a behavioral bug.

**Expected behavior**
The commands should probably call the files _*.dmp_.

**Additional details**
Note that despite the last paragraph about -Fc, the second variant does not use it.

### PR DESCRIPTION
**Pull request by Gauravp-NEC**
Fix for Issue 5545

Fixes #5545

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by Gauravp-NEC** [comment]
I have not updated optional step no 13 & 16. I did not find the file "postgis_restore.pl" for step no. 13 and "postgis.sql" & "spatial_ref_sys.sql" for step no. 16 at location: /usr/share/postgresql/9.4/contrib/postgis-2.1

**Comment by Gauravp-NEC** [comment]
@kowh-ai  @amercader , I have updated the optional step no 13 and 16 in commit https://github.com/ckan/ckan/pull/5567/commits/b3f91d46cb4aeeb4037c94acdb3ca0f5e8387e93 and https://github.com/ckan/ckan/pull/5567/commits/69f6672754fa0ba451785ef3625479ff9b47492d

**Comment by kowh-ai** [comment]
@Gauravp-NEC - I have built the documentation (including your change to `upgrade-postgres.html`) using `sphinx` and it looks good

**Review by kowh-ai — APPROVED** [review]
Updates to the upgrade-postgres.rst file look good and the documentation (sphinx) build is fine

File: doc/maintaining/upgrading/upgrade-postgres.rst [hunk 1]
```diff
@@ -200,29 +201,34 @@ Upgrading
...
 
 
 #. Now restore your databases::
 
-     sudo -u postgres psql --cluster 9.4/main -f backup_ckan.sql
-     sudo -u postgres psql --cluster 9.4/main -f backup_datastore.sql
+     sudo -u postgres pg_restore --cluster 9.5/main -Fc backup_ckan.dmp
+     sudo -u postgres pg_restore --cluster 9.5/main -Fc backup_datastore.dmp
+     
+     If you have create backup remotely in step 12 then:
```
Comment (Chealer) [inline doc/maintaining/upgrading/upgrade-postgres.rst:228]: s/have create/created/

**Pull request merged**